qa/rpc-tests/test_framework/util.py

```diff
@@ -419,3 +419,12 @@ def wait_and_assert_operationid_status(node, myopid, in_status='success', in_err
         return result # if there was an error return the result
     else:
         return txid # otherwise return the txid
+
+# Find a coinbase address on the node, filtering by the number of UTXOs it has.
+# The default cached chain has one address per coinbase output.
+def get_coinbase_address(node, expected_utxos):
```

Comment:
I think it is a little confusing to have to pass in the expected_utxos to get the correct coinbase address. How did this work before when we just called `getnewaddress()` would it give us the either the oldest or the most recent address. Would it be possible to default to that if expected_utxos is not provided or do that instead.

Reply:
In a pairing, @Eirik0 and I decided that we would remove the `expected_utxos` argument, and just return the address that contains the most UTXOs. For all the RPC tests that use this function, this is sufficient (because either they call once targeting a large-UTXO address, or they call every time they need a single UTXO).

Reply:
I instead made it optional, because one of the RPC tests does rely on selecting specific coinbase addresses.